ui/apps/platform/src/types/networkFlow.proto.ts

```diff
@@ -64,6 +64,7 @@ export type ExternalSourceNetworkEntityInfo = {
         name: string;
         cidr?: string;
         default: boolean; // `default` indicates whether the external source is user-generated or system-generated.
+        learned: boolean; // `learned` indicates whether the external source was discovered from network traffic.
```

Comment:
This too, Olivier?

Reply:
:sweat_smile: how is it possible that I forgot this one ! Thanks Mark, I patched it.